src/Web/WebDrapo/package-lock.json

```diff
@@ -0,0 +1,478 @@
+{
```

Comment:
can you remove this file ? i think would be better if we exclude it in gitignore